bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -146,12 +142,14 @@ public void run() {
         // The first invocation of createSslContext() is expensive.
         // Create a throw away object in order to speed up the creation of the first
         // BigtableConnection which uses SslContexts under the covers.
-        try {
-          Class.forName("org.eclipse.jetty.alpn.ALPN");
-        } catch (ClassNotFoundException e1) {
-          LOG.warn(
-            "Could not asynchronously create the ssl context, since ALPN is not installed.");
-          return;
+        if (!OpenSsl.isAvailable()) {
+          try {
+            Class.forName("org.eclipse.jetty.alpn.ALPN");
+          } catch (ClassNotFoundException e1) {
+            LOG.warn(
+              "Could not asynchronously create the ssl context, since ALPN is not installed.");
+            return;
```

Comment:
I was fine with the original logic.  I think that we saw that the gRPC infrastructure's error message was actually less informative in earlier versions of gRPC.  Regardless, I'd rather not change the logic for the alpn-boot case in the tcnative PRs, if that's OK with you.